perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -1378,7 +1382,8 @@ sub process_garden_cancellation : Private {
     $data->{payment_method} = $payment_method;
 
     my $now = DateTime->now->set_time_zone(FixMyStreet->local_time_zone);
-    $c->set_param('Subscription_End_Date', $now->ymd);
+    my $end_date_field = $c->cobrand->call_hook(alternative_backend_field_names => 'Subscription_End_Date') || 'Subscription_End_Date';
+    $c->set_param($end_date_field, $now->ymd);
```

Comment:
Afraid just a tiny bit too much squashing - I think the change to allow Subscription_End_Date to vary should be its own commit, nothing to do with the rest of it :)

Reply:
I've separated this out.